README.md

```diff
@@ -3,18 +3,32 @@
 This is the main website for a pledge to stop supporting Python 2 for free in
 open source software.
 
+# Preview the changes
+
+If you are lazy, just send a PR. The continuous integration will build a preview
```

Comment:
I'd remove the 'lazy' and 'just', and start with "If you send a pull request, Netlify will automatically build a preview for you..."

Then I'd clarify under the *run locally* section that it's fine to use the preview, but if you do want to build the site locally, this is how.